### DISCUSSION TITLE
disable link tooltips?

### DISCUSSION THREAD

**Discussion by jablka** [apache/cordova#440] [General]
I noticed that 
my cordova app (for android) is displaying link tooltips (when hovering links in mobile phone).

(By link I mean html `<a>` elements.)

Is there a way to disable these tooltips?

**Comment by breautek** [comment]
If you mean `<a title="this tooltip">...`

Then I don't think so.

**Reply by jablka** [reply]
no, I don't have `title` attribute in the links...
It seems like cordova generates the tooltip...

**Reply by breautek** [reply]
Cordova is just a packaging tool that packages web assets into a native shell with a webview with a bridge API to allow for JS to communicate to the native environment.

Anything more will be webview/browser features, plugins, or the actual web app.

In this particular case it sounds like a standard browser behaviour however.

**Reply by jablka** [reply]
ok, thanks,
I solved it by replacing `<a>` links with html `onclick` attribute + javascript.
👍
